pipelines/controller/src/main/java/org/openmrs/analytics/metrics/PipelineMetricsEndpoint.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2020-2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openmrs.analytics.metrics;
+
+import io.prometheus.client.Collector.MetricFamilySamples;
+import io.prometheus.client.CollectorRegistry;
```

Comment:
We use these Prometheus specific objects to retrieve the metrics, which otherwise could have been retrieved via `io.micrometer.core.instrument.MeterRegistry`. Only reason to use the Prometheus is that it was easier to extract the metrics.